src/site/stages/build/drupal/assets.js

```diff
@@ -0,0 +1,70 @@
+const getDrupalClient = require('./api');
+
+function replacePathInData(data, replacer) {
+  let current = data;
+  if (Array.isArray(data)) {
+    // This means we're always creating a shallow copy of arrays, but
+    // that seems worth the complexity trade-off
+    current = data.map(item => replacePathInData(item, replacer));
+  } else if (!!current && typeof current === 'object') {
+    Object.keys(current).forEach(key => {
+      let newValue = current;
+
+      if (typeof current[key] === 'string') {
+        newValue = replacer(current[key]);
+      } else {
+        newValue = replacePathInData(current[key], replacer);
+      }
+
+      if (newValue !== current[key]) {
+        current = Object.assign({}, current, {
+          [key]: newValue,
+        });
+      }
+    });
+  }
+
+  return current;
+}
+
+function convertAssetPath(drupalInstance, url) {
+  // After this path are other folders in the image paths,
+  // but it's hard to tell if we can strip them, so I'm leaving them alone
+  const withoutHost = url.replace(`${drupalInstance}/sites/default/files/`, '');
+  const path = withoutHost.split('?', 2)[0];
+
+  // This is sort of naive, but we'd like to have images in the img folder
+  if (
+    ['png', 'jpg', 'jpeg', 'gif', 'svg'].some(ext =>
+      path.toLowerCase().endsWith(ext),
```

Comment:
Not necessary, but [extname](https://nodejs.org/api/path.html#path_path_extname_path) could be nice here.